templates/SilverStripe/CKANRegistry/Page/Layout/CKANRegistryPage.ss

```diff
@@ -0,0 +1,10 @@
+<section class="content">
+    <% if $ElementalArea %>
+        <%-- Support for content blocks, if enabled --%>
+        $ElementalArea
+    <% else %>
+        $Content
+    <% end_if %>
+    <% include CKANRegistry %>
```

Comment:
It might be worth giving this a namespace - I don't see a great reason to have things in a global namespace (of sorts) at this point